app/components/rl-dropdown.js

```diff
@@ -19,13 +18,18 @@ export default Component.extend({
 
     propagateClicks: true,
 
-    manageVisibility: on('didInsertElement', observer('isExpanded', function() {
+    didInsertElement() {
+        this._super(...arguments);
+        this.manageVisibility();
+    },
+
+    manageVisibility: observer('isExpanded', function() {
         if (this.get('isExpanded')) {
             this.$().css('display', 'block');
         } else {
             this.$().css('display', 'none');
         }
-    })),
+    }),
```

Comment:
I think we should replace this entire observer. We could add `style` to `attributeBindings` and bind `display:block/none` to the `isExpanded` property directly.